doc/data/messages/c/consider-using-any-or-all/good.py

```diff
@@ -1 +1,4 @@
-# This is a placeholder for correct code for this message.
+items = (1, 2, 3, 4, 5)
+
+any(item % 2 == 0 for item in items)
```

Comment:
Could you keep the functions from the 'bad' example please ?

Reply:
Do you mean like this?
`any_even = any(item % 2 == 0 for item in items)`?